DbConnection/src/main/java/com/solvd/laba/db/dao/MySQLDAO/MySQLBranchDAO.java

```diff
@@ -0,0 +1,101 @@
+package com.solvd.laba.db.dao.MySQLDAO;
+
+import com.solvd.laba.db.dao.AbstractDAO;
+import com.solvd.laba.db.model.Branch;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+
+public class MySQLBranchDAO extends AbstractDAO<Branch> {
+    private static final Logger logger = LogManager.getLogger(MySQLBranchDAO.class);
+    private final static String GET_BY_ID = "SELECT * FROM branch WHERE id=?";
+    private final static String GET_ALL = "SELECT * FROM branch";
+    private final static String INSERT_VALUES = "INSERT INTO branch (branch_name, branch_location_id) VALUES (?, ?)";
+    private final static String UPDATE_VALUE = "UPDATE branch SET branch_name=?, branch_location_id=? WHERE id=?";
+    private final static String DELETE_VALUE = "DELETE FROM branch WHERE id=?";
+
+    @Override
+    public Branch get(int id) {
+        logger.info("Finding record of ID: " + id + "...");
+        try (PreparedStatement preparedStatement = getConnection().prepareStatement(GET_BY_ID)) {
+            preparedStatement.setInt(1, id);
+            ResultSet result = preparedStatement.executeQuery();
+            if (result.next()) {
+                int branchId = result.getInt("id");
+                String branchName = result.getString("branch_name");
+                int branchLocationId = result.getInt("branch_location_id");
+                return new Branch(branchId, branchName, branchLocationId);
+            } else {
```

Comment:
You don't actually need the else statement; you can just put the body of your else statement after the body of the if statement and the catch blocks. If the code works as expected, the model gets returned, but if it doesn't, it just goes to whatever else is in the body of the method.

Reply:
You are right. Thanks for the tip. I have updated my code.